src/lib/airtable/interface.ts

```diff
@@ -23,13 +23,13 @@ export type Row = {
 export interface UserRecord {
   id: string;
   username: string;
-  email: string;
-  photo?: Airtable.Attachment[];
-  siteIds: string[];
-  sites: SiteRecord[];
-  password: string;
-  name: string;
   admin: boolean;
+  email?: string;
+  photo?: Airtable.Attachment[];
+  siteIds?: string[];
+  sites?: SiteRecord[];
```

Comment:
`sites` should be a `string[]` and we should remove `siteIds`

![image](https://user-images.githubusercontent.com/35501399/114319481-95a8c580-9ac6-11eb-8d47-33d37ba99619.png)


We might have to pass the  User Record we receive from `Login` through schema generator's `transformRecord` since right now the user record is basically what we'd get from an Airtable call. No need to do this in this PR, but we should be aware via a TODO